src/popcorn-list-movies/PopcornListMovies.js

```diff
@@ -66,6 +66,14 @@ export class PopcornListMovies extends LocalizeMixin(
     }
   }
 
+  isMovieImageValid(movie) {
+    if(movie.poster === "N/A") {
+      return "https://ih1.redbubble.net/image.512138487.5983/fposter,small,wall_texture,product,750x1000.u3.jpg"

```

Comment:
If this is going to always be your "placeholder" img, i would download it and save it in the assets folder of your repo.